packages/admin-ui/ui/src/components/forms/product/organize-form/index.tsx

```diff
@@ -86,21 +86,18 @@ const OrganizeForm = ({ form }: Props) => {
         />
       </div>
 
-      {isFeatureEnabled(FeatureFlag.PRODUCT_CATEGORIES) && (
+      {isFeatureEnabled(FeatureFlag.PRODUCT_CATEGORIES) &&
+      categoriesOptions.length === 0 ? (
```

Comment:
todo: Looks like this should check should be flipped? 
```suggestion
      categoriesOptions?.length ? (
```